script.js

```diff
@@ -12,3 +12,83 @@ function clickX() {
   ham.style.display = 'block';
 }
 clickX('');
+
+const popup = document.getElementById('popup');
+const img = document.createElement('img');
+img.src = '/imgs/popup.png';
+popup.appendChild(img);
+
+const he1 = document.getElementById('popup-content');
+const heading = 'Keeping track of \n hundreds of \n components';
+const h1 = document.createElement('h1');
+h1.innerText = heading;
+he1.appendChild(h1);
+
+// Skills Badge List  - For popup
+
+// const ul = document.getElementById('badges');
+// const skills = ['Ruby on Rails', 'CSS', 'JS'];
+// const li = document.createElement('li');
+// for (const skill of skills) {
+//   const text = document.createTextNode(skill);
+//   li.appendChild(text);
+//   ul.appendChild(li);
+// }
+// li.classList.add('badge');
```

Comment:
Kindly take away every commented block of code as they make your code base look a bit messy and less professional😉